src/sentry/tasks/sentry_apps.py

```diff
@@ -306,14 +314,14 @@ def send_webhooks(installation, event, **kwargs):
 
         request_data = AppPlatformEvent(**kwargs)
         send_and_save_webhook_request(
-            servicehook.sentry_app.webhook_url, installation.sentry_app, request_data
+            installation.sentry_app, request_data, servicehook.sentry_app.webhook_url,
```

Comment:
Should we just skip passing the `webhook_url` here?

Reply:
I can't tell if there is a case where `servicehook.sentry_app.webhook_url` and `installation.sentry_app.webhook_url` are different. I assumed there is because the url is explicitly included in the method signature, is it ok to leave it?

Reply:
Ahh... yeah, I have no idea, so probably better to be safe